modules/my_health/app/controllers/my_health/mr_controller.rb

```diff
@@ -5,19 +5,20 @@
 module MyHealth
   class MrController < ApplicationController
     include ActionController::Serialization
-    # include MyHealth::MHVControllerConcerns
+    include MyHealth::MHVControllerConcerns
 
-    skip_before_action :authenticate
+    # skip_before_action :authenticate
 
-    # protected
+    protected
 
     def client
-      @client ||= MedicalRecords::Client.new
+      # @client ||= MedicalRecords::Client.new(session: { user_id: current_user.mhv_correlation_id })
+      @client ||= MedicalRecords::Client.new(session: { user_id: 11_898_795 })
     end
 
-    # def authorize
-    #   # raise_access_denied unless current_user.authorize(:mhv_messaging, :access?)
-    # end
+    def authorize
+      # raise_access_denied unless current_user.authorize(:mhv_messaging, :access?)
```

Comment:
Is this intentionally left commented out?

Reply:
Yes, we have a todo to determine how authorization is going to factor in.